site/content/docs/cicd.md

```diff
@@ -8,17 +8,68 @@ To turn off the interactive features, use the `-s (--silent)` switch. This will
 
     dotnet aws deploy --silent
 
-### Creating a deployment setting file
+### Creating a deployment settings file
 
-To specify the services to deploy and their configurations for your environment, you need to create deployment settings file. The deployment settings file is a JSON configuration file that contains all of the settings that the deployment tool uses to drive the experience. Here is the [JSON file definition](https://github.com/aws/aws-dotnet-deploy/tree/main/src/AWS.Deploy.Recipes/RecipeDefinitions).
+You can persist the deployment configuration to a JSON file using the `--save-settings <SETTINGS_FILE_PATH)>` switch. This JSON file can be version controlled and plugged into your CI/CD system for future deployments.
 
-Storing deployment settings in a JSON file also allows those settings to be version controlled.
+**Note** - The `--save-settings` switch will only persist settings that have been modified (i.e they hold a non-default value). To persist all settings use the `--save-all-settings` switch.
```

Comment:
nit: Internal style guide recommends avoiding Latin abbreviations to better serve a global audience.

Reply:
Fixed